### PR TITLE
1. changed map theme 2. added polygon layer.

### PR DESCRIPTION
**Pull request by Shivam-apna**
changes the map theme to bright .

component preview.

image

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/content/map/map.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <sly data-sly-use.model="io.typerefinery.websight.models.components.Map">
     <div id="map"></div>
```
Comment (sanjayaapna) [inline application/backend/src/main/resources/apps/typerefinery/components/content/map/map.html:2]: don't make unnecessary changes in the file if those are not required.

**Review by sanjayaapna — APPROVED** [review]

File: application/backend/src/main/resources/apps/typerefinery/components/content/map/script.js [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
-      zoom: 0, // starting zoom
-    });
-    map.addControl(new maplibregl.NavigationControl(), "top-right");
-    map.on("error", function (err) {
-      console.log("MAP ERROR", err)
+  const key = 'WTh8FTa6cT027rlVBBbG';
+  const dataId = 'dcb0900e-992a-4ff5-9819-861b4df293e9'
+  const dataId1= 'b70a0972-99a0-4539-98eb-35e04f0c052d'
+  const map = new maplibregl.Map({
+    container: 'map', // container's id or the HTML element in which MapLibre GL JS will render the map
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/map/script.js:6]: This script file will be included even if the map component does not exist in the UI, so please add the condition before executing the functionality,

Check for the map component and then proceed further.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]

**Comment by Shivam-apna** [comment]
please refer https://github.com/innovolve-ai/typerefinery-websight/pull/61 to review the updated changes.

File: application/backend/src/main/resources/apps/typerefinery/components/content/map/script.js (hunk 2 above)
Comment (Shivam-apna) [reply application/backend/src/main/resources/apps/typerefinery/components/content/map/script.js]: resolved.
